internal/resources/resource_volume.go

```diff
@@ -172,6 +178,7 @@ func resourceMetalVolumeCreate(d *schema.ResourceData, meta interface{}) (err er
 		Description: d.Get(vDescription).(string),
 		FlavorID:    vfID,
 		Shareable:   d.Get(vShareable).(bool),
+      WWN:         d.Get
```

Comment:
Remove this line since the client cannot request a WWN.

Reply:
fixed